data/sql/updates/pending_db_world/rev_1680525460717061700.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry`,`ConditionTypeOrReference`, `ConditionValue1`, `ConditionValue2`) VALUES
+(1, 17798, 24313, 7, 197, 1);
```

Comment:
Missing DELETE query and newline at the end

Reply:
There was no previous condition for Kalithresh Normal, but sure